src/stirling/source_connectors/socket_tracer/protocols/types.h

```diff
@@ -49,7 +50,8 @@ using FrameDequeVariant = std::variant<std::monostate,
                                        std::deque<redis::Message>,
                                        std::deque<kafka::Packet>,
                                        std::deque<nats::Message>,
-                                       std::deque<amqp::Frame>>;
+                                       std::deque<amqp::Frame>,
+                                       std::deque<mqtt::Message>>;
```

Comment:
Changes to this file should also come in a later PR (when the MQTT trace bpf test is added).